sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-pre-validation-service.ts

```diff
@@ -36,15 +36,18 @@ export class ECertPreValidationService {
    * time in an attempt to advise students that the disbursement will be blocked if no action is taken.
    * The validations are executed in the next pending disbursement.
    * @param applicationId applications to have its disbursements validated. If no disbursement is pending
+   * @param allowNonCompleted: only select completed applications or allow any status.
    * no validations will be executed.
```

Comment:
I think you should swap the lines 39 and 40.

Reply:
fixed